tasks/drop.lasso.yml

```
@@ -0,0 +1,55 @@
+---
+- name: "drop.lasso: Install periodic script"
+  ansible.builtin.template:
+    dest: /usr/local/etc/periodic/daily/900.drop-lasso
+    src: "{{ pf_drop_lasso_template }}"
+    owner: root
+    group: wheel
+    mode: "0755"
+    # keeping a backup will result in the script being run multiple times
+    backup: no
+  when: pf_drop_lasso_enable | bool
+  
+- name: "drop.lasso: Enable periodic script"
+  community.general.sysrc:
+    name:  daily_pf_droplasso_enable
+    value: "{{ pf_drop_lasso_enable | bool | ternary('YES', 'NO') }}"
+    path:  "{{ pf_drop_lasso_periodic_file }}"
+
+- name: "drop.lasso: Enable v4"
+  community.general.sysrc:
+    name:  daily_pf_droplasso_v4
+    value: "{{ pf_drop_lasso_v4 | bool | ternary('YES', 'NO') }}"
+    path:  "{{ pf_drop_lasso_periodic_file }}"
+  when: pf_drop_lasso_enable | bool
+
+- name: "drop.lasso: Enable v6"
+  community.general.sysrc:
+    name:  daily_pf_droplasso_v6
+    value: "{{ pf_drop_lasso_v6 | bool | ternary('YES', 'NO') }}"
+    path:  "{{ pf_drop_lasso_periodic_file }}"
+  when: pf_drop_lasso_enable | bool
+
+- name: "drop.lasso: v4 file"
+  community.general.sysrc:
+    name: daily_pf_droplasso_v4_filename
+    # this variable is also set in templates/default-drop-lasso.j2
+    value: "{{ pf_drop_lasso_drop_v4_filename | default('/etc/pf.drop_v4.txt') }}"
+    path:  "{{ pf_drop_lasso_periodic_file }}"
+  when: pf_drop_lasso_drop_v4_filename is defined
+
+- name: "drop.lasso: v6 file"
+  community.general.sysrc:
+    name: daily_pf_droplasso_v6_filename
+    # this variable is also set in templates/default-drop-lasso.j2
+    value: "{{ pf_drop_lasso_drop_v6_filename | default('/etc/pf.drop_v6.txt') }}"
+    path:  "{{ pf_drop_lasso_periodic_file }}"
+  when: pf_drop_lasso_drop_v6_filename is defined
```

Apply *pf_drop_lasso_rcconf* in a loop

```suggestion
- name: "drop.lasso: Configure periodic."
  community.general.sysrc:
    path: "{{ pf_drop_lasso_periodic_file }}"
    name: "{{ item.name }}"
    value: "{{ item.value }}"
  loop: "{{ pf_drop_lasso_rcconf }}"
```